source/_integrations/bmw_connected_drive.markdown

```diff
@@ -138,32 +192,53 @@ If you do not want this, trigger the `vehicle_finder` action from your phone and
 On some older cars (non i3/i8 series produced before 7/2014) this action will fail in getting your vehicles position, if the vehicle is more than 1.5 km away from the location of your Home Assistant instance. This is a limitation of the BMW API.
 {% endnote %}
 
-## Selects
+### Selects
 
 If you have a (PH)EV, you can control the charging process through Home Assistant. The selects are created automatically depending on your vehicle's capabilities and can be pressed/executed from the UI or using the `select.select_option` action. For more information, please see the [select documentation](/integrations/select/).
 
-Using these selects will impact the state of your vehicle. Use them with care!
-
 - **Charging Mode**: Vehicle can be set to `IMMEDIATE_CHARGING` (charge as soon as plugged in) or `DELAYED_CHARGING` (charge only if within charging window). It can be used to start/stop charging if the charging window is set accordingly.
 - **AC Charging Limit**: The maximum current a vehicle will charge with. Not available on all EVs.
 
-## Switches
+### Switches
 
 If supported by your vehicle, you can display and toggle remote actions with start/stop functionality.
 
-Using these selects will impact the state of your vehicle, use them with care!
-
-- **Climate**: Toggle vehicle climatization. It is not possible to force it to heating/cooling, the vehicle will decide on its own. If turned on, it will run for 30 minutes (as if toggled via the MyBMW app).
+- **Climate**: Toggle vehicle climatization. It is not possible to force it to heat or cool; the vehicle will decide on its own. If turned on, it will run for 30 minutes (as if toggled via the MyBMW app).
 - **Charging**: Toggle vehicle charging if plugged in. Only available on some electric vehicles.
 
-## Numbers
+### Numbers
 
 If you have a (PH)EV, you can control the charging process through Home Assistant. The number entities are created automatically depending on your vehicle's capabilities and can be changed from the UI or using the `number.set_value` action. For more information, please see the [number documentation](/integrations/number/).
 
-Using these selects will impact the state of your vehicle, use them with care!
-
 - **Target SoC**: Vehicle will charge until this battery level is reached. Not available on all EVs.
 
+## Troubleshooting
+{% details "Problem: Invalid authentication" %}
+
+This can happen during initial login or after some time. Please do the following steps:
+- Log in to your MyBMW **website** and verify your credentials (i.e. ensure that username and password are correct).
```

Comment:
```suggestion
- Log in to your MyBMW **website** and verify your credentials (for example, ensure that username and password are correct).
```
following MS Style Guide, we [avoid e.g.](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/eg)